bliss/conf/base_config.yaml

```diff
@@ -67,13 +67,17 @@ cached_simulator:
     batch_size: ${generate.batch_size} # 128 for large cached
     num_workers: 0
     cached_data_path: ${generate.cached_data_path}
+    train_n_batches: null
     file_prefix: ${generate.file_prefix}
+    bands: [0, 1, 2, 3, 4]
     val_split_file_idxs: null # inclusive range
     test_split_file_idxs: null # inclusive range
+    z_score_transform: True
 
 encoder:
     _target_: bliss.encoder.Encoder
-    bands: [2]
+    z_score: True
```

Comment:
Similar to other comment, do we still need this here?

Reply:
I wanted to keep this one optional in case a user wants to do single-band experiments with or without z-scoring. I have it set in encoder so that if multiple bands are specified in encoder, `self.z_score` is automatically set to `True` regardless of what is specified in the config. If single band is specified, the config specifiation is used instead.